pool/pool.py

```diff
@@ -626,7 +646,7 @@ async def add_farmer(self, request: PostFarmerRequest, metadata: RequestMetadata
                 last_state,
                 uint64(0),
                 difficulty,
-                request.payload.payout_instructions,
+                puzzle_hash,
```

Comment:
We either need to convert to hex string here and below or we need to adjust the type of it in `FarmerRecord` to `bytes32` and convert accordingly to string in `SqlitePoolStore` (or just store it as blob there?). I think i prefer the latter.